crates/levm/src/vm.rs

```diff
@@ -115,6 +132,7 @@ pub struct VM {
     pub accrued_substate: Substate,
     /// Mapping between addresses (160-bit identifiers) and account
     /// states.
+    // pub state: WorldState,
```

Comment:
I see that we are keeping the `Db` over the `WorldState`, I think the choice is fine, but if so we should drop the `WorldState` abstraction so there is no confusion

Reply:
True